### PR TITLE
Artp 595/storybook trade notification

### PR DESCRIPTION
**Pull request by snony**

### CODE REVIEW

File: src/client/src/shell/notification/TradeNotification.stories.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { action } from '@storybook/addon-actions'
-import { select } from '@storybook/addon-knobs'
+import { capitalize } from 'lodash'
 import { storiesOf } from '@storybook/react'
 import React from 'react'
 import { Story } from 'rt-storybook'
 import { styled } from 'rt-theme'
 import TradeNotification, { Props } from './TradeNotification'
+import {TradeStatus} from 'rt-types'
```
Comment (james-gray-adaptive) [inline src/client/src/shell/notification/TradeNotification.stories.tsx]: Minor: Formatting `{ Trade Status }`

File: src/client/src/shell/notification/TradeNotification.stories.tsx [hunk 2]
```diff
@@ -38,19 +32,32 @@ const Centered = styled('div')`
...
-stories.add('Default', () => (
-  <Story>
-    <Centered>
-      <NotificationContainer>
-        <TradeNotification {...props} />
-      </NotificationContainer>
-    </Centered>
-  </Story>
-))
+const getPropsByStatus:(status:TradeStatus)=>Props = (status)=>(
```
Comment (james-gray-adaptive) [inline src/client/src/shell/notification/TradeNotification.stories.tsx]: Minor: Formatting

File: src/client/src/shell/notification/TradeNotification.stories.tsx (hunk 1 above)
Comment (snony) [reply src/client/src/shell/notification/TradeNotification.stories.tsx]: I don't understand your comment, TradeStatus has been formatted correctly and it is a type.

File: src/client/src/shell/notification/TradeNotification.stories.tsx (hunk 2 above)
Comment (snony) [reply src/client/src/shell/notification/TradeNotification.stories.tsx]: The formatting is correct on my machine.

**Pull request merged**